src/SDKs/Network/Management.Network/Generated/Models/IssueType.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Network.Models
+{
+    using Azure;
+    using Management;
+    using Network;
+
+    /// <summary>
+    /// Defines values for IssueType.
+    /// </summary>
+    public static class IssueType
```

Comment:
It's an enum for type in ConnectivityIssue.cs

Here is spec in swagger:
        "type": {
          "readOnly": true,
          "type": "string",
          "enum": [ "Unknown", "AgentStopped", "GuestFirewall", "DnsResolution", "SocketBind", "NetworkSecurityRule", "UserDefinedRoute", "PortThrottled", "Platform" ],
          "x-ms-enum": {
            "name": "IssueType",
            "modelAsString": true
          },
          "description": "The type of issue."
        },